src/components/ContentDecoder.tsx

```diff
@@ -34,6 +42,11 @@ export function ContentDecoder({ to, chainId = '1', descriptionResult, analyzeRe
                 url={url}
             />
             <Styled.Results>
+                {analyzeResult.transaction_simulation && shouldShowSimulation(analyzeResult.transaction_simulation) && (
```

Comment:
nit: two checks feel redundant, include 'analyzeResult.transaction_simulation' in shouldShowSimulation